Dockerfile

```diff
@@ -44,16 +44,18 @@ RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
 RUN . "$NVM_DIR/nvm.sh" && nvm use v${NODE_VERSION}
 RUN . "$NVM_DIR/nvm.sh" && nvm alias default v${NODE_VERSION}
 ENV PATH="/root/.nvm/versions/node/v${NODE_VERSION}/bin/:${PATH}"
-RUN npm i -g yarn
 
 # Install soroban-cli
 COPY --from=soroban-cli /usr/local/cargo/bin/soroban $CARGO_HOME/bin/
 
+# Install phantomjs
+RUN apt -y install phantomjs
```

Comment:
do you think this would ever regress to need a specific version of phantom, is it worth wiring it in now as another build arg with 'latest' as default.

Reply:
Yeah, weirdly whenever I try to `apt install phantomjs@2.1.1` it says the package can't be found. But also that's the version that is installed, so...